src/modules/services/components/bulk/AssignServiceButton.tsx

```diff
@@ -77,23 +103,66 @@ const AssignServiceButton: React.FC<Props> = ({
     return 'Enroll + Assign';
   }, [client.activeEnrollment, isAssignedOnDate, localDisabled]);
 
+  const startIcon =
+    isAssignedOnDate && !localDisabled ? (
+      <CheckIcon />
+    ) : clientAlerts.length > 0 ? (
+      <WarningAmberRoundedIcon />
+    ) : undefined;
+
   return (
     <>
       <ButtonTooltipContainer title={disabledReason} placement='top-start'>
         <LoadingButton
           onClick={onClick}
           loading={loading}
           disabled={disabled || localDisabled}
-          startIcon={
-            isAssignedOnDate && !localDisabled ? <CheckIcon /> : undefined
-          }
+          startIcon={startIcon}
           fullWidth
           variant={isAssignedOnDate ? 'contained' : 'gray'}
         >
           {buttonText}
         </LoadingButton>
       </ButtonTooltipContainer>
       {apolloError && <ApolloErrorAlert error={apolloError} />}
+      <CommonDialog fullWidth open={clientAlertDialogOpen}>
+        <DialogTitle>
+          {clientAlerts.length === 1 ? 'Client Alert' : 'Client Alerts'}
+        </DialogTitle>
+        <DialogContent sx={{ mt: 2 }}>
+          <ClientAlertStack clientAlerts={clientAlerts} />
+        </DialogContent>
+        <DialogActions>
+          <FormActions
+            onDiscard={() => setClientAlertDialogOpen(false)}
+            onSubmit={() => {
+              bulkAssign({
+                variables: {
+                  input: {
+                    ...queryVariables,
+                    clientIds: [client.id],
+                  },
+                },
+                onCompleted: () => setLocalDisabled(true),
+              });
+              setClientAlertDialogOpen(false);
+            }}
+            config={[
+              {
+                action: FormActionTypes.Discard,
+                id: 'discard',
+                label: `Cancel ${serviceTypeName}`,
+                buttonProps: { variant: 'gray' },
+              },
+              {
+                action: FormActionTypes.Submit,
+                id: 'submit',
+                label: `Add ${serviceTypeName}`,
```

Comment:
Awaiting question about button color in [figma](https://www.figma.com/file/F9EHR2IZYuMAlOJ9aOG2IL?type=design&node-id=3417-24669&mode=design#782287629)

Reply:
Resolved per figma thread - button should be client main color, which it is by default.